source/ways-of-working/environments/external-access.html.md.erb

```diff
@@ -0,0 +1,81 @@
+
+[Traefik](https://traefik.io/traefik/) is used as the ingress class in HMCTS applications.
+
+The following traefik instances are available to use:
+
+| IngressClass | Default | Application configuration |
+|-|-|-|
+| traefik | Yes | Application will be reachable externally without VPN. End user will be required to authenticate to Azure AD before being redirected to the application |
+| traefik-private | No | Application will be available over VPN only |
+| traefik-no-proxy | No | Application will be reachable externally without VPN or authentication |
+
+### Protecting applications with Azure AD authentication
+
+If you do not set an ingressClass flag in code, it will default to traefik. 
+    
+To ensure end users must authenticate to Azure AD before they can view the application in their web browser, you must also set the `enableOAuth` flag to `true`
+
+```yaml
+nodejs:
+    ingressClass: traefik
+    disableTraefikTls: false
+    enableOAuth: true
+```
+
+The authentication with Azure AD is handled by an Azure AD App Registration in the HMCTS Demo Azure AD tenant. To allow end users to be authenticated to your application, you must add a redirect URI for your application to the Azure AD App Registration.
+
+*If you are not currently a member of the HMCTS Demo Azure AD tenant, check with your team to see if anyone else is. Guests in this tenant have permission to invite other guests.*
+
+There are separate Azure AD App Registrations for [CFT](https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationMenuBlade/Authentication/appId/fbf4cb6e-f09d-4fc4-89fa-b94cb582cb18/isMSAApp/) and [SDS](https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationMenuBlade/Authentication/appId/2a3f8b5a-ec0e-470d-b979-fc79d3e74cba/isMSAApp/).
+
+Navigate to the appropriate app registration, click on `Authentication` and click `Add URI` under `Redirect URIs`.
+
+Your redirect URI should be the full URI of your application appended with `/oauth-proxy/callback` e.g. https://toffee.demo.platform.hmcts.net/oauth-proxy/callback
+
+### Protecting applications with HMCTS VPN
+
+If you want your application to only be available on VPN, you must specify the `traefik-private` ingressClass.
+
+You do not need to specify the `enableOAuth` flag in this case.
+
+```yaml
+nodejs:
+    ingressClass: traefik-private
+    disableTraefikTls: false
+```
+
+### Making applications externally reachable without VPN or authentication
+
+If you want your application to only be available without the need to connect to VPN and without the need to authenticate to Azure AD, you must specify the `traefik-no-proxy` ingressClass.
+
+You do not need to specify the `enableOAuth` flag in this case.
+
+```yaml
+nodejs:
+    ingressClass: traefik-no-proxy
+    disableTraefikTls: false
+```
+
+This should only be done for applications that do not work with Azure AD authentication or VPN.
```

Comment:
This should be right under the heading and called out with a warning like done in https://hmcts.github.io/ways-of-working/new-component/secrets-management.html#secrets-management